cli/src/base/AccountsCommandBase.ts

```diff
@@ -224,6 +224,16 @@ export default abstract class AccountsCommandBase extends ApiCommandBase {
     }
   }
 
+  async getRequiredMemberId(): Promise<number> {
+    const account = await this.getRequiredSelectedAccount()
+    const memberIds = await this.getApi().getMemberIdsByControllerAccount(account.address)
+    if (!memberIds.length) {
+      this.error('Membership required to access this command!', { exit: ExitCodes.AccessDenied })
+    }
+
+    return memberIds[0].toNumber() // FIXME: Temporary solution (just using the first one)
```

Comment:
I wanted to raise this issue in previous PR about member id selection and clearly you are aware of it hence your "FIXME" comment. Just wanted to assert that yes it would be good to give use option to select the member id of their choice when there are multiple matches..